package.json

```diff
@@ -70,6 +70,7 @@
     "@babel/core": "^7.6.4",
     "@babel/plugin-proposal-class-properties": "^7.5.5",
     "@babel/plugin-transform-modules-commonjs": "^7.6.0",
+    "@babel/plugin-transform-object-assign": "^7.8.3",
```

Comment:
just to confirm - this won't polyfill, it is just ponyfill?

Reply:
It looks fine: https://babeljs.io/docs/en/next/babel-plugin-transform-object-assign.html#example

Reply:
Yes